src/Uno.UI.Runtime.Skia.Gtk/Input/GtkKeyboardInputSource.cs

```diff
@@ -49,15 +49,15 @@ private void OnKeyPressEvent(EventKey evt)
 				this.Log().Trace($"OnKeyPressEvent: {evt.Key} -> {virtualKey}");
 			}
 
-			InputHelper.TryConvertKeyCodeToScanCode(evt.HardwareKeycode, out var scanCode);
```

Comment:
`TryConvertKeyCodeToScanCode` is probably unused now and can be removed?

Reply:
@Youssef1313 Yeah. And I will remove it.